tests/system/download_diagnostics.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import logging.config
+
+logging.config.fileConfig('logging.conf')
+logger = logging.getLogger(__name__)
+
+import os # NOQA E402
+import time # NOQA E402
+from shakedown.clients import node # NOQA E402
+
+
+def main(download_dir):
+    client = node.Node()
+
+    bundle = client.diagnostics.create()
+
+    while bundle.status() is None:
+        logger.info('Diagnostic bundle is not complete yet. Waiting another 10 seconds.')
+        time.sleep(10)
+
+    bundle.download(download_dir)
```

Comment:
why don't we pass the output of bundle.status() in here and we query the status once more in that method...

Reply:
That's a fair point. We should pass in the download URL.